source/linux/background-managers/swaybg.js

```diff
@@ -0,0 +1,9 @@
+import {commandExists, spawn} from '../util.js';
+
+export async function isAvailable() {
+	return commandExists('swaybg');
+}
+
+export async function set(imagePath) {
+	await spawn('swaybg', ['-i', imagePath, '-m', 'fill']);
```

Comment:
Any why can't you use the existing `execFile`?

Reply:
Because swaybg must work constantly to set wallpaper. When I use execFile the serWallpaper function never completes due to which any code written after the function is not executed.

Reply:
Looks like you can use `swaymsg`.

https://github.com/search?q=swaymsg+swaybg&type=code

Reply:
For swaymsg you must install sway but you can use swaybg without sway

Reply:
Alright. https://github.com/sindresorhus/wallpaper/pull/93/files#diff-96da661640c7b1ea27ced9841f06a8dab68749a29c97cd37b39ead556530da9fR7 should be moved back into this file. It doesn't make sense as a generic utility.